packages/asset-buyer/package.json

```diff
@@ -34,21 +34,26 @@
     "bugs": {
         "url": "https://github.com/0xProject/0x-monorepo/issues"
     },
-    "homepage": "https://github.com/0xProject/0x-monorepo/packages/forwarder-helper/README.md",
+    "homepage": "https://github.com/0xProject/0x-monorepo/packages/asset-buyer/README.md",
     "dependencies": {
-        "@0xproject/assert": "^1.0.9",
-        "@0xproject/json-schemas": "^1.0.2",
-        "@0xproject/order-utils": "^1.0.3",
-        "@0xproject/types": "^1.0.2",
-        "@0xproject/typescript-typings": "^2.0.1",
-        "@0xproject/utils": "^1.0.9",
-        "@types/node": "*",
+        "@0xproject/assert": "^1.0.8",
```

Comment:
There has been a publish in the mean-time. This should be `^1.0.9` and others are probably also out-of-date.